src/main/java/seedu/address/model/ModelHistory.java

```diff
@@ -0,0 +1,23 @@
+package seedu.address.model;
+
+import seedu.address.commons.exceptions.AlfredModelHistoryException;
+import seedu.address.model.entitylist.MentorList;
+import seedu.address.model.entitylist.ParticipantList;
+import seedu.address.model.entitylist.TeamList;
+
+/**
+ * API for ModelHistoryManager
+ */
+public interface ModelHistory {
```

Comment:
Just wondering, why would you need to take extra parameteres pListId or XListId in general? Since it can be gotten from plist.getLastUsedId() haha 😄

Reply:
This is because the participantID can keep incrementing after a certain state (say, state number x). So when i undo up to state number x, if i do not update the static ID attribute to the value at state x, the static ID attribute would be incorrect